### PR TITLE
2 5 whats changed summary

### PR DESCRIPTION
**Pull request by nancyhild**
Right now this summary article can stand alone--the next step is to update the relevant articles and add links from this summary to the articles.

### CODE REVIEW

File: docs/beaver-builder/about-this-release.md [hunk 1]
```diff
@@ -2,54 +2,133 @@
...
-The **Advanced** tab is documented [here](/beaver-builder/layouts/advanced-tab-rows-columns-modules.md).
-* Added SpinupWP to the [cache clearing tool](/beaver-builder/troubleshooting/debugging/cache-clearing-tool.md).
+* There is basic support for the official [Font Awesome Plugin](https://wordpress.org/plugins/font-awesome/) so you can use that plugin's shortcodes in your Beaver Builder layouts. If you want to use Font Awesome Pro icons, be sure to follow the plugin installation instructions for enabling your domain in the Font Awesome CDN.
+* There's now an easy way to disable [smooth scrolling](/beaver-builder/advanced-builder-techniques/smooth-scrolling-links.md) so a link jumps to its target on the same page.  
+on the **Advanced** tab where you set the anchor in the **ID** field, add `fl-no-scroll` to the **Class** field  below it. See the [smooth scrolling article](/beaver-builder/advanced-builder-techniques/smooth-scrolling-links.md) for more information about creating target anchors on a page and linking to them.
+* Change in Debug Mode operation: errors are output only if  the`showerrors` parameter is in the URL.
```
Comment (nancyhild) [inline docs/beaver-builder/about-this-release.md:132]: This debug item is written in developerese. Does the user have to add this parameter to show errors--or remove the parameter to not show errors? Or is this just offering a reason why they might not see errors when they expect to? Why is this an improvement over how it worked before?

File: docs/beaver-builder/about-this-release.md [hunk 2]
```diff
@@ -2,54 +2,133 @@
...
+There are two additional options on the **Structured data** tab for `contentUrl` and `embedUrl`.
 
-Three modules have been added to the free version of Beaver Builder:
+## Column improvements
 
-* [Heading](/beaver-builder/layouts/modules/heading.md)
-* [Icon](/beaver-builder/layouts/modules/icon-and-icon-group.md)
-* [Button](/beaver-builder/layouts/modules/button/button.md)
+* You can now set the width for single columns.  
+![Price Table module 2.5 features](/img/2-5-features-single-column-width.png)
```
Comment (Dannyholt) [inline docs/beaver-builder/about-this-release.md:116]:
```markdown
![Price Table module 2.5 features](/img/2-5-features-single-column-width.png) 
```
Alt text is incorrect. :)

File: docs/beaver-builder/about-this-release.md [hunk 3]
```diff
@@ -2,54 +2,133 @@
...
 
-Three modules have been added to the free version of Beaver Builder:
+## Column improvements
 
-* [Heading](/beaver-builder/layouts/modules/heading.md)
-* [Icon](/beaver-builder/layouts/modules/icon-and-icon-group.md)
-* [Button](/beaver-builder/layouts/modules/button/button.md)
+* You can now set the width for single columns.  
+![Price Table module 2.5 features](/img/2-5-features-single-column-width.png) 
+<!-- * There are now custom positions available for backgrounds. -->
```
Comment (Dannyholt) [inline docs/beaver-builder/about-this-release.md:117]:
```markdown
<!-- * There are now custom positions available for backgrounds. -->
```
Is this supposed to be commented out?

File: docs/beaver-builder/about-this-release.md (hunk 1 above)
Comment (Dannyholt) [reply docs/beaver-builder/about-this-release.md:132]: Works in the same way as `nohistory` meaning after enabling BB debug mode then in your site URL you add `/?showerrors`

If the URL already has `?` being used such as when using the BB debug URL i.e:

```markup
mysite.com?fldebug=ed52f356
```

Then you need to use `&showerrors` like so:

```markup
mysite.com?fldebug=ed52f356&showerrors
```

File: docs/beaver-builder/about-this-release.md (hunk 3 above)
Comment (nancyhild) [reply docs/beaver-builder/about-this-release.md:117]: Yes because I couldn't find it in the UI--if it's there will you tell me where it is?

**Comment by nancyhild** [comment]
Abandoning this pull request.

**Pull request closed without merging**